src/Acuminator/Acuminator.Utilities/DiagnosticSuppression/SuppressionManagerInitInfo.cs

```diff
@@ -25,15 +25,16 @@ public SuppressionManagerInitInfo(string path, bool generateSuppressionBase)
 			: false;
 
 		public bool Equals(SuppressionManagerInitInfo other) => 
-			GenerateSuppressionBase == other.GenerateSuppressionBase && Path == other.Path;
+			GenerateSuppressionBase == other.GenerateSuppressionBase && 
+			string.Equals(Path, other.Path, StringComparison.OrdinalIgnoreCase);
 
 		public override int GetHashCode()
 		{
 			int hash = 17;
 
 			unchecked
 			{
-				hash = 23 * hash + (Path?.GetHashCode() ?? 0);
+				hash = 23 * hash + (Path?.ToUpper().GetHashCode() ?? 0);
```

Comment:
`ToUpperInvariant`

Reply:
done. Still it's a very very low probability that someone will have Acuminator suppresion file with turkish name